jpa/hibernate4_1/src/main/java/org/jboss/as/jpa/hibernate4/infinispan/SharedInfinispanRegionFactory.java

```diff
@@ -24,62 +24,16 @@
 
 import java.util.Properties;
 
-import org.hibernate.cache.CacheException;
-import org.infinispan.AdvancedCache;
-import org.infinispan.manager.EmbeddedCacheManager;
-import org.jboss.as.jpa.hibernate4.HibernateSecondLevelCache;
-import org.jipijapa.cache.spi.Classification;
-import org.jipijapa.cache.spi.Wrapper;
-import org.jipijapa.event.impl.internal.Notification;
-
 /**
- * Infinispan-backed region factory that retrieves its cache manager from the Infinispan subsystem.
- * This is used for (JPA) container managed persistence contexts.
- * Each deployment application will use a unique Hibernate cache region name in the shared cache.
+ * Disabled 2lc.
  *
  * @author Paul Ferraro
  * @author Scott Marlow
  */
 public class SharedInfinispanRegionFactory extends InfinispanRegionFactory {
```

Comment:
There seems to be some removal of functionality in general with Hibernate 4.3 support. I'll stop commenting on each thing I find, but we need an explanation on why this was removed :)